scripts/release.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This is only being run from the Procfile, which means it's only relevant to Heroku, right? If that's right, the name `release.sh` seems misleading. I would think this script creates a new release or something like that. Any reason we wouldn't name it something like `heroku-db-cache-setup.sh`? If there's more that might go in here, maybe something more general like `heroku-setup.sh`?

Reply:
I'll name it `heroku-release-phase.sh` since that's specifically what it's for.